admin_manual/appliance/enterprise-trial.rst

```diff
@@ -0,0 +1,25 @@
+The appliance has the community edition of ownCloud but can be easily upgraded to the enterprise edition. This upgrade gives you access to a Free-30-Day-Trial of the enterprise edition and all it's features. All you need is an email address to get started. Here are the necessary steps:
+
+- Visit https://marketplace.owncloud.com/enterprise-trial
+
+- Enter your email address and chose a password
+
+- Click on Complete Process
+
+- Check your email and and activate your account
+
+- log in with your credentials at marketplace.owncloud.com
+
+- copy the API key
+
+Now you have to go to your ownCloud and enter the Market app
```

Comment:
"go to your ownCloud and enter the Market app". Should this be "go to your ownCloud installation and enable the Market app"?

Reply:
yes